.gitignore

```diff
@@ -2,3 +2,6 @@ MANIFEST
 .DS_Store
 _book
 node_modules
+.idea
+.swp
+
```

Comment:
Is this empty line needed?

Reply:
It's recommended by Git

Reply:
Do you have a source for this? https://git-scm.com/docs/gitignore doesn't mention anything.

Reply:
oh, sorry, i made a mistake, it doesn't matter.

Reply:
Could you remove this?

Reply:
removed https://github.com/ondrejsika/djangogirls-tutorial/commit/c67bebc9f8a11294ae8019bc0d30fa42dcb5c87b